pkg/profiles/profiles.go

```diff
@@ -372,8 +372,12 @@ func Validate(data []byte) error {
 
 	rb := serviceProfile.Spec.RetryBudget
 	if rb != nil {
-		if rb.MinRetriesPerSecond == 0 || rb.RetryRatio == 0 || rb.TTL == "" {
-			return fmt.Errorf("ServiceProfile \"%s\" RetryBudget missing fields (requires minRetriesPerSecond, retryRatio, ttl)", serviceProfile.Name)
+		if rb.RetryRatio < 0 {
+			return fmt.Errorf("ServiceProfile \"%s\" RetryBudget RetryRatio must be greater than zero: %f", serviceProfile.Name, rb.RetryRatio)
```

Comment:
Nit on the error message: "RetryBudget RetryRatio must be greater than or equal to zero:" or "etryBudget RetryRatio must be non-negative:"